ajax/libs/unveil2/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "unveil2",
+  "filename": "jquery.unveil2.min.js",
+  "description": "A very lightweight jQuery plugin to lazy load images",
+  "homepage": "https://nabble.github.io/unveil2/",
+  "author": "nabble",
```

Comment:
Let's follow the info from the upstream: `Joram van den Boezem`.
(https://github.com/nabble/unveil2/blob/master/package.json#L9)